backend/ml/inputs.py

```diff
@@ -41,63 +46,75 @@ def ratings_properties(self) -> pd.DataFrame:
             * `user_id`: int
             * `entity_id`: int or str
             * `is_public`: bool
-            * `is_trusted`: bool
-            * `is_supertrusted`: bool
+            * `is_scaling_calibration_user`: bool
+            * `trust_score`: float
         """
         raise NotImplementedError
 
 
 class MlInputFromPublicDataset(MlInput):
```

Comment:
It's not clear to me what's the use case for MlInputFromPublicDataset.

Shouldn't we have more of a standalone interface that also downloads the dataset and unzips it without having to find the right URL ? Or is MlInputFromPublicDataset just the low-level component and we are going in the future to add a user-friendly standalone script that will use it ? Will MlInputFromPublicDataset have any use for people that want to work with notebooks ?

Reply:
The idea would be to use it as a low-level component in the future yes. It's not really usable for now as running Mehestan requires to write the results into the Django database, but I think it would make sense to introduce another abstraction on top. Something like `TournesolRun(input_mode, output_mode, ml_parameters)` which would enable to run the algorithms without any dependence to Django or PostgreSQL. Possibly from notebooks, but the exact use-case has not been specified yet.


But that was not really the purpose of this branch :wink: I'm sorry to have brought this topic on this PR.
Maybe we'd better discuss that separately. I can even delete this class `MlInputFromPublicDataset` completely and move it to a new branch if that can help.

Reply:
I don't really care which PR or branch we do these corrections, that's fine. And if it's just a lower-level abstraction that's ok too, we'll think later about how to make a high-level practical interface..